CONTRIBUTORS.txt

```diff
@@ -49,13 +50,16 @@ needs to be listed here.
 - John Connolly <connoljo2@gmail.com>
 - John Holdsworth <github@johnholdsworth.com>
 - Jovan Milenkovic <jovan.milenkovic@htecgroup.com>
+- Julian Lettner <yln@users.noreply.github.com>
```

Comment:
@yln if you would like me to add a proper email address, please let me know which one.

Reply:
done